docs/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-openfeign-async-docs</artifactId>
+	<version>0.1.0-SNAPSHOT</version>
```

Comment:
`0.0.1-SNAPSHOT` ?

Reply:
I don't think so. In x.y.z, z is a patch release. I think 0.1.0 is fine to start.